frontend/src/queries/nodes/DataNode/dataNodeLogic.ts

```diff
@@ -250,6 +253,10 @@ export const dataNodeLogic = kea<dataNodeLogicType>([
                         })
                         breakpoint()
                         actions.setElapsedTime(response.duration)
+                        if ('query' in query) {
+                            actions.setCachedQuery(query.query as string)
+                            actions.setCachedResponse(response.data)
```

Comment:
I'm not too familiar with how the cache works but how does it affect memory usage in the browser? Will it copy the data leading to a doubling of memory usage? If so, how large can the responses be, and could it cause performance issues?

Reply:
hmm fair point. This caching isn't 100% necessary as the improvements in this PR should properly use the remote cache on /query that we have. So, I'll rework this to not use for now